sdk/src/main/java/com/sygic/travel/sdk/auth/utils/AuthUtils.kt

```diff
@@ -0,0 +1,42 @@
+package com.sygic.travel.sdk.auth.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+
+
+object AuthUtils {
```

Comment:
This class is wrong, we want to have proper injectable authUtils.

- pass shared preferences as constructor dependency
- this dependency is created by koeind, context is passed as runtime argument into codein
- also the name should be different from "AuthPreferences", something like "SygicTravel.auth"?

Reply:
:heavy_check_mark: